src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -463,6 +447,29 @@ public Nsi getNsiByNsiId(
             @PageableDefault(sort = {"crn"}, direction = Sort.Direction.ASC) final Pageable pageable) {
         return offenderService.getAllPrimaryIdentifiers(filter, pageable);
     }
+
+
+    @ApiOperation(value = "Return custodial status information by crn, convictionId and sentenceId")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+                    @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
+                    @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
+                    @ApiResponse(code = 404, message = "The offender CRN is not found", response = ErrorResponse.class),
+                    @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+            })
+    @GetMapping(path = "/offenders/crn/{crn}/convictions/{convictionId}/sentences/{sentenceId}/custodialStatus")
+    public CustodialStatus getCustodialStatusBySentenceId(
+            @ApiParam(name = "crn", value = "CRN for the offender", example = "A123456", required = true)
+            @NotNull @PathVariable(value = "crn") final String crn,
+            @ApiParam(name = "convictionId", value = "ID for the conviction / event", example = "2500295345", required = true)
+            @NotNull @PathVariable(value = "convictionId") final Long convictionId,
```

Comment:
Do we need @NotNull on path variables?

Reply:
No don't see what effect it would have. Will remove.